apps/meteor/app/voip/server/startup.ts

```diff
@@ -14,6 +14,8 @@ settings.changeMultiple(
 	(_values) => {
 		// Here, if 4 settings are changed at once, we're getting 4 diff callbacks. The good part is that all callbacks are fired almost instantly
 		// So to avoid stopping/starting voip too often, we debounce the call and restart 1 second after the last setting has reached us.
-		return debouncedRefresh();
+		if (settings.get('VoIP_Enabled')) {
+			return debouncedRefresh();
+		}
```

Comment:
changeMultiple is already debounced, so there is no need for the line 6

Reply:
![image](https://user-images.githubusercontent.com/5263975/182952171-e824aff1-bd8a-4272-8dad-ba0f2e7d0b41.png)

Reply:
![image](https://user-images.githubusercontent.com/5263975/182952251-0ace095b-6128-4e4c-a79e-1af21436f3eb.png)

Reply:
the debounce was higher than the settings' default (1000ms while settings' is 500ms), but I agree 500ms should be enough, so removed.

this comment from your image was not being used as documentation for editor's intellisense, so I changed that as well

Reply:
![image](https://user-images.githubusercontent.com/5263975/182985411-dc72f9de-1da8-4e08-acf9-1cca02b53e0a.png)
its possible to change the default if 1s is the exactly value